Dockerfile

```diff
@@ -10,6 +10,7 @@ WORKDIR /app
 COPY --from=build-image ./app/dist ./dist
 COPY package* ./
 COPY migrations ./migrations
-RUN npm ci --production
+COPY migrateUpWithWrapper.mjs ./migrateUpWithWrapper.mjs
```

Comment:
Hackish patch: this wrapper runs `node-pg-migrate` like it is currently in use in this repository but makes sure that when an error is sent to `stederr` that the db password is hidden from the logs.